Creating the research page and its content.

Cydonia01 opened the pull request:
Created the layout of research page and added some sample content. Also, added tags to the faculty members to represent them in the page of a research field.

doganulus commented:
Top-level Research Categories for this task:

- Artificial Intelligence (EmU, CeS, TuG)
- Bioinformatics (ArÖ, İnB)
- Computer Systems (ArY, CaÖ, AlŞ, DoU)
- Computer Vision and Pattern Recognition (LaA, BeG)
- Computers and Society (SuÜ, DiG, PıY, AAS)
- Machine Learning (İnB, BeG, ATC)
- Computation and Language (TuG, SuÜ, ArÖ, OnG)
- Networking and Internet Architecture (CeE, FaA, BiY, TuT, AtÖ, CaT, ÇaS)
- Software Engineering (SuÜ, AyB, DoU)
- Parallel, Distributed, and Cloud Computing (TuT, CaÖ, AtÖ, CeE, AlŞ, ÇaS)
- Robotics and Control (EmU, DoU, ArY)
- Theory and Algorithms (CeS, DoU, AlŞ, CaÖ)

Cydonia01 commented:
Added 12 research areas and people tags. Also, added a paragraph to each area to explain the area briefly. The paragraphs are AI generated. If they are OK, then I will leave them like that.
